tailwind.js

```diff
@@ -43,7 +43,7 @@ View the full documentation at https://tailwindcss.com.
 */
 const chroma = require('chroma-js');
 
-const BASE_FONT_SIZE_PX = 14;
+const BASE_FONT_SIZE_PX = 16;
```

Comment:
Why did you decide to move to 16? Does the calculation below not need to be changed?

Reply:
since the rem value scales in the css we would have to make this value scale as well. So to avoid alot of potentially unnecessary work I set it to the max base font size